django/tasks/constants.py

```diff
@@ -0,0 +1,4 @@
+UNKNOWN = 0
+PENDING = 1
+FAILED = 2
+SUCCESS = 3
```

Comment:
I thought about that too, though I'm not sure it's worth a dependency on Python <3.4.